packages/contracts/contracts/interpreters/ETHInterpreter.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity 0.5.8;
+pragma experimental "ABIEncoderV2";
+
+
+contract ETHInterpreter {
+
+  struct ETHTransferWrapper {
+    ETHTransfer[] wrapped;
+  }
+
+  struct ETHTransfer {
+    address to;
+    uint256 amount;
+  }
+
+  struct Param {
+    uint256 limit;
+  }
+
+  function interpret(
+    bytes memory input, bytes memory params
+  ) public {
+
+    ETHTransfer[] memory transfers =
+      abi.decode(input, (ETHTransferWrapper)).wrapped;
+    uint256 limitRemaining = abi.decode(params, (Param)).limit;
+
+    for (uint i=0; i<transfers.length; i++) {
+      address payable to = address(uint160(transfers[i].to));
+      uint256 amount = transfers[i].amount;
+
+      require(amount <= limitRemaining, "hit the limit");
+      limitRemaining -= amount;
+
+      // note: send() is deliberately used instead of transfer() here
```

Comment:
Instead of saying it is deliberately used, explain _why_ it is deliberately used.